locale/fr_CA/locale.xml

```diff
@@ -953,12 +935,102 @@ La validation peut échouer sur les URL de recherche qui nécessitent des formul
+	<message key="about.onlineSubmissions.newSubmission">Faire une nouvelle soumission</message>
+	<message key="about.onlineSubmissions.viewSubmissions">Voir vos soumissions en attente</message>
+	<message key="help.goToEditPage">Éditer {$sectionTitle}</message>
+	<message key="user.role.journalAssistant">Assistant</message>
+	<message key="user.role.journalAssistants">Assistants</message>
+	<message key="user.role.guestEditor">Rédacteur invité</message>
+	<message key="user.role.guestEditors">Rédacteurs invités</message>
+	<message key="installer.preInstallationInstructions"><![CDATA[<h4>Étapes précédant l'installation</h4>
+
+<p>1. Les fichiers et répertoires suivants (et leur contenu) doivent être accessibles en écriture :</p>
+<ul>
+	<li><tt>config.inc.php</tt> est accessible en écriture (facultatif) : {$writable_config}</li>
+	<li><tt>public/</tt> est accessible en écriture : {$writable_public}</li>
+	<li><tt>cache/</tt> est accessible en écriture : {$writable_cache}</li>
+	<li><tt>cache/t_cache/</tt> est accessible en écriture : {$writable_templates_cache}</li>
+	<li><tt>cache/t_compile/</tt> est accessible en écriture : {$writable_templates_compile}</li>
+	<li><tt>cache/_db</tt> est accessible en écriture : {$writable_db_cache}</li>
+</ul>
+
+<p>2. Un répertoire pour stocker les fichiers téléchargés doit être créé et accessible en écriture (voir "Paramètres des fichiers" ci-dessous).</p>]]></message>
+	<message key="article.pluginMissing"><![CDATA[*<p>Le fichier PDF sélectionné devrait se s'afficher ici si un plugiciel de lecture de PDF est installé dans votre navigateur (par exemple, une version récente de <a href="https://get.adobe.com/reader/">Adobe Acrobat Reader</a>).</p>]]></message>
+	<message key="manager.setup.clockssDescription"><![CDATA[Open Journal Systems prend également en charge le système <a href="http://clockss.org/" target="_blank"> CLOCKSS </a> (Controlled Lots of Copies Keep Stuff Safe) pour assurer un archivage sécurisé et permanent pour le journal. CLOCKSS est basé sur le logiciel open source LOCKSS, développé à la bibliothèque de l'Université de Stanford, qui permet aux bibliothèques de préserver des revues Web en sondant régulièrement les sites Web de revues enregistrées pour les contenus nouvellement publiés, pour ensuite les archiver. Chaque archive est continuellement validée par rapport à d'autres caches et si le contenu est corrompu ou perdu, les autres caches ou le journal sont utilisés pour la restaurer.
+<br /> <br />
+L'installation de CLOCKSS pour Open Journal Systems n'a pas besoin d'avoir lieu avant que la publication soit en cours.]]></message>
+	<message key="manager.setup.citations.enable">Activer l'Assistant au balisage de référence (Citation Markup Assistant)</message>
+	<message key="submission.layout.galley.remoteURL">URL du contenu hébergé à distance</message>
+	<message key=""><![CDATA[]]></message>
```

Comment:
Please remove empty lines (probably created by translator plugin) here and in the next two lines.